locales/en/translation.json

```diff
@@ -88,6 +87,7 @@
   "Carb Ratios": "Carb Ratios",
   "Carbohydrates": "Carbohydrates",
   "Carbs": "Carbs",
+  "Cartridge expiration": "Cartridge expiration",
```

Comment:
why don't you re-use the existing translation ?
"Pump cartridge expiration date"
```suggestion
```